### ISSUE TITLE
Can't update or reinstall to 5.0.0

### ISSUE THREAD

**Issue by magpie514** [PrestonKnopp/language-gdscript#27]
I tried removing and clearing cache folders, but I keep getting this error while trying to install.
System is Ubuntu 19.10 64b, using Atom 1.42.0.
```
Installing “lang-gdscript@5.0.0” failed.Hide output…

> tree-sitter-gdscript@0.2.5 install /tmp/apm-install-dir-1191126-18961-1x8wj2y.698w/node_modules/lang-gdscript/node_modules/tree-sitter-gdscript
> prebuild-install || node-gyp rebuild


prebuild-install WARN install No prebuilt binaries found (target=4.2.7 runtime=electron arch=x64 libc= platform=linux)
Aborted
Aborted
gyp: Call to 'node -e "require('nan')"' returned exit status 134 while in binding.gyp. while trying to load binding.gyp
gyp ERR! configure error 
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (/usr/share/atom/resources/app/apm/node_modules/npm/node_modules/node-gyp/lib/configure.js:345:16)
gyp ERR! stack     at ChildProcess.emit (events.js:182:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:237:12)
gyp ERR! System Linux 5.3.0-24-generic
gyp ERR! command "/usr/share/atom/resources/app/apm/bin/node" "/usr/share/atom/resources/app/apm/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /tmp/apm-install-dir-1191126-18961-1x8wj2y.698w/node_modules/lang-gdscript/node_modules/tree-sitter-gdscript
gyp ERR! node -v v10.2.1
gyp ERR! node-gyp -v v3.7.0
gyp ERR! not ok 
Aborted
npm WARN enoent ENOENT: no such file or directory, open '/tmp/apm-install-dir-1191126-18961-1x8wj2y.698w/package.json'
npm WARN ws@7.2.1 requires a peer of bufferutil@^4.0.1 but none is installed. You must install peer dependencies yourself.
npm WARN ws@7.2.1 requires a peer of utf-8-validate@^5.0.2 but none is installed. You must install peer dependencies yourself.
npm WARN apm-install-dir-1191126-18961-1x8wj2y.698w No description
npm WARN apm-install-dir-1191126-18961-1x8wj2y.698w No repository field.
npm WARN apm-install-dir-1191126-18961-1x8wj2y.698w No README data
npm WARN apm-install-dir-1191126-18961-1x8wj2y.698w No license field.

npm ERR! code ELIFECYCLE
npm ERR! errno 134
npm ERR! tree-sitter-gdscript@0.2.5 install: `prebuild-install || node-gyp rebuild`
npm ERR! Exit status 134
npm ERR! 
npm ERR! Failed at the tree-sitter-gdscript@0.2.5 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/magpie/.atom/.apm/_logs/2019-12-26T22_24_14_587Z-debug.log
```

**Comment by PrestonKnopp** [comment]
Hi, thanks for posting an issue. I'm sorry about the late response. If you are still looking for a fix, read on, otherwise let me know and I'll close this issue.

Can you try removing the package and then reinstalling with `apm install lang-gdscript`. Additionally, can you upload the debug log, too.

I vaguely remember a similar issue that was remedied by manually installing `node-gyp`:

`npm install -g node-gyp`

Looks like you will have to manually install the websocket peer dependencies:

`npm install -g bufferutil utf-8-validate`

**Comment by magpie514** [comment]
I tried installing ws (and the two dependencies) manually, as well as reinstalling gyp, but something is not behaving well with npm in general. JS is not a language I use for myself, so I don't have the drive to fix it unless I really need it later on (I might just wipe it out and reinstall during some holidays).
So, I've decided to move to vscode/vim/godot's own editor depending on what area I'm working on, I pretty much only use atom for some parts of the github integration nowadays.

Because of that, I think it'll be better to close the issue unless someone else has a similar problem, but thanks for your help regardless!
